### PR TITLE
[Feature] Running Profile OLAP_SCAN_NODE layering and enhance readability

### LINKED ISSUE #4821: [Feature] Running Profile OLAP_SCAN_NODE node has poor readability
**Is your feature request related to a problem? Please describe.**
In the Running Profile, the statistical values of the OLAP_SCAN_NODE node are too many, including the statistical values of OlapScanner and SegmentIterator.
This leads to poor readability of the Running Profile and contains multiple meaningless statistical values.

**Describe the solution you'd like**
I suggest dividing the OLAP_SCAN_NODE profile into three layers: ʻOLAP_SCAN_NODE`,  ʻOlapScanner`, and `SegmentIterator` according to the calling relationship, and delete meaningless statistical values.

### PR DESCRIPTION
**Pull request by xinyiZzz**
## Proposed changes

mainly includes:
- `OLAP_SCAN_NODE` profile layering: `OLAP_SCAN_NODE`,`OlapScanner`, and `SegmentIterator`.
- Delete meaningless statistical values. mainly in scan_node.cpp.
- Increase `RowsConditionsFiltered` statistical, split from `RowsDelFiltered`, the meaning is the number of rows filtered by various column indexes, only in segment V2.
- Modify the document based on the above, and enhance readability.

## Types of changes

- [x] Documentation Update (if none of the other choices apply)
- [x] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

- [x] I have create an issue on (Fix #4821 ), and have described the bug/feature there in detail
- [x] If this change need a document change, I have updated the document

### CODE REVIEW

**Review by HappenLee — CHANGES_REQUESTED** [review]

**Comment by xinyiZzz** [comment]
@HappenLee Good question, I have moved `_segment_profile` and `scanner_profile` to ʻOlapScanNode` and modified the document.

File: docs/zh-CN/administrator-guide/running-profile.md [hunk 1]
```diff
@@ -147,103 +147,74 @@ BE端收集的统计信息较多，下面列出了各个参数的对应含义：
...
-    * IOTimer 是 IO 时间，能够直接反映 IO 操作耗时。这里是所有 Scanner 线程累加的 IO 时间。
-    * NumScanners 表示 Scanner 线程数。线程数过多或过少都会影响查询效率。同时可以用一些汇总指标除以线程数来大致的估算每个线程的耗时。
-    * TabletCount 表示需要扫描的 tablet 数量。数量过多可能意味着需要大量的随机读取和数据合并操作。
-    * UncompressedBytesRead 间接反映了读取的数据量。如果该数值较大，说明可能有大量的 IO 操作。
-    * CachedPagesNum 和 TotalPagesNum。对于 V2 格式，可以查看命中 PageCache 的情况。命中率越高，说明 IO 和解压操作耗时越少。
+  - `OlapScanner` 下的很多指标，如 `IOTimer`，`BlockFetchTime` 等都是所有 Scanner 线程指标的累加，因此数值可能会比较大。并且因为 Scanner 线程是异步读取数据的，所以这些累加指标只能反映 Scanner 累加的工作时间，并不直接代表 ScanNode 的耗时。ScanNode 在整个查询计划中的耗时占比为 `Active` 字段记录的值。有时会出现比如 `IOTimer` 有几十秒，而 `Active` 实际只有几秒钟。这种情况通常因为：
+    - `IOTimer` 为多个 Scanner 的累加时间，而 Scanner 数量较多。
+    - 上层节点比较耗时。比如上层节点耗时 100秒，而底层 ScanNode 只需 10秒。则反映在 `Active` 的字段可能只有几毫秒。因为在上层处理数据的同时，ScanNode 已经异步的进行了数据扫描并准备好了数据。当上层节点从 ScanNode 获取数据时，可以获取到已经准备好的数据，因此 Active 时间很短。
+  - `NumScanners` 表示 Scanner 线程数。线程数过多或过少都会影响查询效率。同时可以用一些汇总指标除以线程数来大致的估算每个线程的耗时。
```
Comment (HappenLee) [inline docs/zh-CN/administrator-guide/running-profile.md:214]: NumScanner应该表示的是提交到线程池Task数目，并不能直接推导出他的线程数

**Review by HappenLee — APPROVED** [review]

**Comment by HappenLee** [comment]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**